ni_measurementlink_generator/tests/test_assets/example_renders/SampleMeasurementWithAnnotations.serviceconfig

```diff
@@ -0,0 +1,16 @@
+{
+  "services": [
+    {
+      "displayName": "Sample Measurement With Annotations",
+      "serviceClass": "SampleMeasurement_Python",
+      "descriptionUrl": "https://www.example.com/SampleMeasurement.html",
+      "providedInterfaces": [ "ni.measurementlink.measurement.v1.MeasurementService" ],
+      "path": "start.bat",
+      "annotations": {
+        "ni/service.description": "Measurement description",
+        "ni/service.collection": "Measurement collection",
+        "ni/service.tags": ['M1', 'M2', 'M3']
```

Comment:
JSON strings must be double quotes, not single.

Reply:
Fixed